ds-sklearn/setup.md

```diff
@@ -0,0 +1 @@
+Please follow [these setup instructions](https://github.com/INRIA/scikit-learn-mooc/blob/main/local-install-instructions.md)
```

Comment:
I prefer using jupyterlab. Shall we add it?

Reply:
Yes, me too. But I don't think we need to add it, it will only confuse people, it will be installed with anaconda anyways.